src/VSCodeExtension/src/generator-qsharp/.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
If you're adding an editor config file, I'd suggest doing so at the root of the repo rather than in a subfolder.

Reply:
Actually, this came from the generator-generator scaffolding, so I didn't actually intend to add it. I'll remove it.